spyder/utils/environ.py

```diff
@@ -12,33 +12,74 @@
 import os
 
 # Third party imports
-from qtpy.QtWidgets import QDialog, QMessageBox
+from qtpy.QtWidgets import QMessageBox
+try:
+    import winreg
+except ImportError:
```

Comment:
```suggestion
except Exception:
```

This could fail for reasons other than `ImportError` which would lead Spyder to crash at startup.